tests/bearlib/languages/documentation/DocumentationExtractionTest.py

```diff
@@ -87,12 +88,12 @@ def test_extract_documentation_CPP(self):
 
         self.assertEqual(tuple(extract_documentation(data, "CPP", "doxygen")),
                          (DocumentationComment(
-                              ("\n"
-                               " This is the main function.\n"
-                               " @returns Exit code.\n"
-                               "          Or any other number.\n"),
-                              docstyle_CPP_doxygen.markers[0],
-                              TextRange.from_values(4, 1, 8, 4)),
+                             ("\n"
+                              " This is the main function.\n"
+                              " @returns Exit code.\n"
+                              "          Or any other number.\n"),
+                             docstyle_CPP_doxygen.markers[0],
+                             TextRange.from_values(4, 1, 8, 4)),
                           DocumentationComment(
```

Comment:
right right I now realize that I have changed too much.
Just a few lines need fixing... :+1: